contracts/event/MultipleResultsEvent.sol

```diff
@@ -496,20 +446,19 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         require(value > 0, "Vote amount should be > 0");
 
         // Update balances
-        _eventRounds[_currentRound].balances[resultIndex].total =
-            _eventRounds[_currentRound].balances[resultIndex].total.add(value);
-        _eventRounds[_currentRound].balances[resultIndex].bets[from] =
-            _eventRounds[_currentRound].balances[resultIndex].bets[from].add(value);
-        _resultTotals[resultIndex] = _resultTotals[resultIndex].add(value);
         _totalBets = _totalBets.add(value);
+        _voteRoundsTotals[resultIndex] = _voteRoundsTotals[resultIndex].add(value);
+        _voteRoundsUserTotals[from][resultIndex] =
+            _voteRoundsUserTotals[from][resultIndex].add(value);
+        _currentVotingRoundTotals[resultIndex] = 
+            _currentVotingRoundTotals[resultIndex].add(value);
 
         // Emit events
         emit VotePlaced(address(this), from, resultIndex, value, _currentRound);
 
         // If voted over the threshold, create a new DecentralizedOracle round
-        uint resultVotes = _eventRounds[_currentRound].balances[resultIndex].total;
-        uint threshold = _eventRounds[_currentRound].consensusThreshold;
-        if (resultVotes >= threshold) {
+        if (_currentVotingRoundTotals[resultIndex] >=
+            _eventRounds[_currentRound].consensusThreshold) {
             voteSetResult(from, resultIndex, value);
```

Comment:
This is a single called method, would you like to inline this method?

Reply:
prefer not too. keeps the set result logic separate and readable. and not sure if the vote + voteSetResult might make the stack too deep.